### ISSUE TITLE
html <br> tags not working in code block version

### ISSUE THREAD

**Issue by ksdavidc** [NicoNekoru/obsidan-advanced-table-xt#43]
## This works

````

| head 1<br />c1 | head 1<br />c1 |
|:-----------------:|:-----------------:|
|    r1 <br>c1    |     r1 <br>c2     |
|         ^         |        rr         |
|         ^         |        44         |


````

## This doesn't 

````
```sheet
| head 1<br>c1   | head 1<br>c1|
| :---:       | :---:     |
| r1 <br>c1 | r1 <br>c2 |
| ^           | rr        |
| ^           | 44        |
|                   |
```

````

### Raw

<img width="533" alt="Screen Shot 2023-12-17 at 3 27 11 AM" src="https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/62230004/d1f86190-0aeb-4e7f-99b8-922d0acbeec9">

### Rendered

<img width="273" alt="Screen Shot 2023-12-17 at 3 28 27 AM" src="https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/62230004/155ef33f-3e98-4402-bf6f-af3856f8d2d6">

**Comment by zhoulonghao520** [comment]
I have also encountered this bug before, uninstalling and reinstalling the plugin once should resolve the issue.

**Comment by ddazbo** [comment]
Ive just started having this issue as well, most likely after I updated to the latest version. The <br> tag had worked reliably so far.  Unfortunately @zhoulonghao520 's suggestion didn't work for me

**Comment by NicoNekoru** [comment]
Should be fixed #52
